deps/src/main/java/com/microsoft/azure/sdk/iot/deps/serializer/MethodParser.java

```diff
@@ -167,14 +167,14 @@ public synchronized void fromJson(String json) throws IllegalArgumentException
             JsonElement jsonElement = jsonParser.parse(json);
             if (jsonElement instanceof JsonPrimitive || jsonElement instanceof JsonArray)
             {
-                /**
-                 *  Basic JSON String or Array
-                 *  Ex:
-                 *      "this is payload."
-                 *      1
-                 *      true
-                 *      2.0e10
-                 *      2.6
+                /*
```

Comment:
These documentation changes fix "Dangling Javadoc comment inspection":
Reports dangling Javadoc comments. Javadoc comment are dangling if they don't belong to any class, method or field. For example a Javadoc comment in between method declarations that have their own javadoc comments.